evolver/tests/test_logging.py

```diff
@@ -20,7 +20,11 @@ def test_log_capture_handler():
     assert hist.kind == "log"
 
 
-def test_log_evolver_hookup():
+def test_log_evolver_hookup(caplog):
```

Comment:
pytest was capturing the log and not passing it on, but we specifically want it for this test.